src/torrijas.md

```diff
@@ -0,0 +1,36 @@
+# Torrijas
+
+![Torrijas](pix/torrijas.webp)
+
+Torrijas are a spanish dessert traditionally eaten during the Holy Week (Easter).
+It is said they were first cooked using the leftovers from the Lent.
+
+- ⏲️ Prep time: 1 min
+- 🍳 Cook time: 5 min
+
+## Ingredients
+
+- Dried bread (or Torrijas bread). Must be bread from a regular loaf, not from a tin loaf.
+- Milk (to soak the bread, 2 cups per 16 torrijas apporx.)
+- Eggs (to coat the bread, 2 eggs per 16 torrijas apporx.)
+- Cinnamon
+- Grated lemon rind (optional)
+- Sugar
+- Olive oil
+
+## Directions
+
+1. Mull the milk with cinnamon and a bit of sugar (not too much, the milk is already sweet). Add the lemon rind if you want.
+2. Cool down the milk.
+3. While the milk is cooling, beat the eggs and heat up half a pan of olive oil.
+4. Slice the dry bread in loaves. Soak it in the milk.
+5. Coat the bread soaking it in the beaten eggs.
+6. Fry the soaked & coated bread with the heated oil.
+7. Optionally sprinkle some sugar onto the Torrijas.
+8. Enjoy when they are colled down. Don't eat too much, it won't be easy resisting!
+
+## Contribution
+
+- **danoloan** - [webpage](danoloan.es)
```

Comment:
```suggestion
- **danoloan** - [webpage](https://danoloan.es)
```

Need the `http://` protocol specified, otherwise markdown will look for `danoloan.es` locally on the based.cooking website